TombEditor/Controls/Panel3D/ToolHandler.cs

```diff
@@ -12,7 +12,7 @@ private class ToolHandler
         {
             private class ReferenceCell
             {
-                public readonly short[,] Heights = new short[2, 4] { { 0, 0, 0, 0 }, { 0, 0, 0, 0 } };
+                public readonly short[,] Heights = new short[9, 4] { { 0, 0, 0, 0 }, { 0, 0, 0, 0 }, { 0, 0, 0, 0 }, { 0, 0, 0, 0 }, { 0, 0, 0, 0 }, { 0, 0, 0, 0 }, { 0, 0, 0, 0 }, { 0, 0, 0, 0 }, { 0, 0, 0, 0 } };
```

Comment:
Is there a more easy way to store zeroed 2D array in C#?

Reply:
It seems like all the zeros can be removed completely, since 0 is the default value for short afaik. Will fix.